pkg/controller/config/config.go

```diff
@@ -18,12 +18,13 @@ type Config struct {
 	ProfilerPort          config.Port     `json:"prof-port" pflag:"\"10254\",Profiler port"`
 	FlinkIngressURLFormat string          `json:"ingressUrlFormat"`
 	UseProxy              bool            `json:"useKubectlProxy"`
-	ProxyPort             config.Port     `json:"ProxyPort" pflag:"\"8001\",The port at which flink cluster runs locally"`
+	ProxyPort             config.Port     `json:"proxyPort" pflag:"\"8001\",The port at which flink cluster runs locally"`
 	ContainerNameFormat   string          `json:"containerNameFormat"`
 	Workers               int             `json:"workers" pflag:"4,Number of routines to process custom resource"`
 	BaseBackoffDuration   config.Duration `json:"baseBackoffDuration" pflag:"\"100ms\",Determines the base backoff for exponential retries."`
 	MaxBackoffDuration    config.Duration `json:"maxBackoffDuration" pflag:"\"30s\",Determines the max backoff for exponential retries."`
 	MaxErrDuration        config.Duration `json:"maxErrDuration" pflag:"\"5m\",Determines the max time to wait on errors."`
+	PrometheusPort        config.Port     `json:"prometheusPort" pflag:"\"9249\",The port at which Flink metrics are exported from job manager and task manager pods"`
```

Comment:
This would affect all the application in the cluster. Not a fan of setting configs at operator layer when it can be configured at individual application layer.